.github/workflows/release.yml

```diff
@@ -3,20 +3,40 @@ on:
   push:
     branches:
       - master
+      - beta
       - "[0-9]+.[0-9]+.x"
 
+permissions:
+  contents: read # for checkout
```

Comment:
Is this a global setting for any operation?

Reply:
Correct, this sets up the minimum permissions for the entire workflow. Individual jobs can then specify the permissions they need individually.